tools/cuttlefish-host-image-installer/version_parser.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2018 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0(the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Parse changelog and get either latest version or stable version.
+
+"""
+
+import os.path
+import re
+import sys
+
+def main():
+    stable_version = False
+    if len(sys.argv) < 2 or len(sys.argv) > 3:
+        # Exit non-zero if input parameter format is wrong
+        print("length is wrong")
```

Comment:
Print a "usage" string instead, something like `version_parser.py <changelog> [latest|stable]`

Reply:
Input format is wrong.
usage: version_parser.py <changelog> [latest|stable]
return the corresponding version and write the corresponding changelog context in current directory